cypress.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
+  e2e: {
+    baseUrl: 'https://staging.deriv.com',
```

Comment:
@habib-deriv I have a question, we have baseUrl in e2e but when we want to access it you use Cypress.env is this okay? 

`url = Cypress.env('base_url') || 'https://staging.deriv.com'`